### ISSUE TITLE
Error when building ros2_object_analytics.

### ISSUE THREAD

**Issue by Schloern93** [intel/ros2_object_analytics#119]
Hello,

While building the project I get an error in 3 packages.
I have attached the terminal output

Many thanks in advance
[Error_building_OA.txt](https://github.com/intel/ros2_object_analytics/files/5596733/Error_building_OA.txt)

**Comment by askokostic** [comment]
Having the same issue. @Schloern93, did you fix it?

**Comment by Schloern93** [comment]
No, unfortunately not. I did my object detection using the ROS package "YOLO ROS: Real-Time Object Detection for ROS". Then I created a sorted Point Cloud message with the following ROS package.“ depth image proc“
I did the object detection by assigning the found objects of the YOLO-ROS package (in pixel) and the points of the PointCloud message (points x/y/z coordinates).
I hope this can help you.


Von: askokostic
Gesendet: Mittwoch, 17. März 2021 11:43
An: intel/ros2_object_analytics
Cc: Schloern93; Mention
Betreff: Re: [intel/ros2_object_analytics] Error when building ros2_object_analytics. (#119)

Having the same issue. @Schloern93, did you fix it?
